source/includes/usage-examples/run-command.c

```diff
@@ -0,0 +1,157 @@
+#include <bson/bson.h>
+#include <mongoc/mongoc.h>
+#include <stdio.h>
+
+int
+main (int argc, char *argv[])
+{
+    mongoc_client_t *client;
+    mongoc_collection_t *collection;
+    char *str;
+    bson_error_t error;
+
+    mongoc_init ();
+
+    client =
+        mongoc_client_new ("<connection string URI>");
+    collection = mongoc_client_get_collection (client, "<database name>", "collection name");
+
+    {
+        // start-execute-command
+        bson_t *command, reply;
+        bson_error_t error;
+        bool retval;
+    
+        command = BCON_NEW ("hello", BCON_INT32 (1));
```

Comment:
Consider moving declarations to the line first-used to shorten examples.

C89 required variables to be declared at the start of a scope, but the C driver does not support C89.

```c
// start-execute-command
bson_t reply;
bson_error_t error;

bson_t *command = BCON_NEW ("hello", BCON_INT32 (1));
bool retval = mongoc_client_command_simple (client, "admin", command, NULL, &reply, &error);
```

Comment applies to other declarations. Added suggestion to [C/C++ Driver Code Example Patterns and Style Reference](https://docs.google.com/document/d/1wBsUnT69a7XNTUOZd1agQae_pL1fREl-mcg34QgYaCs/edit#heading=h.j6clrnd6ap6d)

Reply:
Ah I see what you mean. Updating.